paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -58,8 +57,9 @@ import kotlin.io.path.invariantSeparatorsPathString
 class PaparazziPlugin : Plugin<Project> {
   override fun apply(project: Project) {
     val legacyResourceLoadingEnabled = (project.findProperty("app.cash.paparazzi.legacy.resource.loading") as? String)?.toBoolean() ?: false
+    val legacyAssetLoadingEnabled = (project.findProperty("app.cash.paparazzi.legacy.asset.loading") as? String)?.toBoolean() ?: false
 
-    if (legacyResourceLoadingEnabled) {
+    if (legacyResourceLoadingEnabled || legacyAssetLoadingEnabled) {
```

Comment:
I'm assuming we want to do an `||` rather than `&&` here. But honestly I'm not sure why we need this if-statement at all.

Reply:
interesting, originally this if/else block was designed with only resources in mind, so I missed updating this with assets.

that being said, if one were to run legacy resources more, but modern assets mode, I'm not sure how that would behave...

Reply:
There is a reason given how we validate which modules are supported.  Legacy mode only support library modules, whereas modern supports application/dynamic features.